js/everything.js

```diff
@@ -580,10 +584,16 @@ $(function() {
     $("#form").submit(function(event) {
         event.preventDefault();
         var formValue = $.trim($("#v").val());
-        var formValueTime = /[?&]t=(\d*)/g.exec(formValue);
-        if (formValueTime && formValueTime.length > 1) {
-            formValueTime = parseInt(formValueTime[1], 10);
-            formValue = formValue.replace(/&t=\d*$/g, "");
+        var formValueTime = /[?&]t=(\d+)|[?&]time_continue=(\d+)/g.exec(formValue);
+        if (formValueTime && formValueTime.length > 2) {
```

Comment:
I think we can simplify this regex to`/[?&](t|time_continue)=(\d*)/g`. The difference with mine is we say "we'll take either `t` or `time_continue` and use the first one we find". Doing this should allow us to simplify the logic below

Play with it: https://regex101.com/r/VaRFn0/3

Reply:
Sure, looks good, @shakeelmohamed . Will use that one. One thing is that I think it should be `/[?&](t|time_continue)=(\d+)/g`, i.e. `\d+` instead of `\d*` to match atleast one digit following the `=` sign, so that it works for cases like `?time_continue=&t=2` properly. Of course those are invalid cases, but we may want to handle them correctly?

Reply:
@hchhabra oh yes you're right about `\d+`, I was playing with that regex quickly. Go for it!

Reply:
Made the change, @shakeelmohamed .